src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -299,6 +299,25 @@ def get_request_status(
             detail="Cannot specify both succeeded and failed query params.",
         )
 
+    for end, start, field_name in [
+        [created_lt, created_gt, "created"],
+        [completed_lt, completed_gt, "completed"],
+        [errored_lt, errored_gt, "errorer"],
+        [started_lt, started_gt, "started"],
+    ]:
+        if end is None or start is None:
+            continue
+        if not (isinstance(end, datetime) and isinstance(start, datetime)):
+            continue
+        else:
+            if end < start:
+                # With date fields, if the start date is after the end date, return a 400
+                # because no records will lie within this range.
+                raise HTTPException(
```

Comment:
how does front end handle 4xx for date range slider? Is this just an error banner?

Reply:
IIRC it currently doesn't, but the request won't work either way in this scenario currently (as no requests can be returned).